driver-core/src/test/java/com/datastax/driver/core/DataTypeTest.java

```diff
@@ -43,19 +42,66 @@ static boolean exclude(DataType t) {
         return t.getName() == DataType.Name.COUNTER;
     }
 
-    private static String[] getCQLStringTestData(DataType dt) {
+    private static String[] getOutputCQLStringTestData(DataType dt) {
```

Comment:
I don't like the fact that `getOutputCQLStringTestData`, `getInputCQLStringTestData` and `getTestData` are 3 different methods, but the data they return must match. I'm going to refactor that with a private class.